### PR TITLE
Several updates and Issue Closures...

### PR DESCRIPTION
**Pull request by Raistlfiren**
**Closes issues -** 

- https://github.com/FAForever/website/issues/30
- https://github.com/FAForever/website/issues/29
- https://github.com/FAForever/website/issues/28

**Summary of commits -** 

1.  Added Amazon Buy Now button 
2.  Added Status link to footer 
3. Configured challonge username and apikey as env variables
4. Added social media links
5. Added FA icons

### CODE REVIEW

**Comment by Raistlfiren** [comment]
@Blodir bump! :)

**Comment by Blodir** [comment]
- Why was the download button on the navbar unseparated and style changed?
- Footer seems to be broken, I see "connect" without the icons. Also we can replace "Unofficial Tools" with "Connect". There's already some social media links on the footer, they should also be removed with the addition of icons

**Comment by Raistlfiren** [comment]
Thanks for going over this PR. I fixed the issues that you ran into. You will need to run `grunt serve:dev` to recompile the SASS files for font awesome.

File: templates/layouts/default.jade [hunk 1]
```diff
@@ -58,21 +57,21 @@ html
...
 							li.dropdown
-								a(href="#" class="dropdown-toggle" data-toggle="dropdown" role="button") Documentation 
+								a(href="#" class="dropdown-toggle" data-toggle="dropdown" role="button") Documentation
 									span.caret
 									ul.dropdown-menu(role="menu")
 										li: a(href="http://wiki.faforever.com/index.php?title=Main_Page") Wiki
 										li: a(href="http://www.faforever.com/faf/unitsDB/") Unit Database
 							li: a(href='/contribution') Contribute
 						ul.nav.navbar-nav.navbar-right
-							li: a(href='http://content.faforever.com/FAF.msi' id="btn-dl-nav") Download Client
```
Comment (Blodir) [inline templates/layouts/default.jade]: Revert this

File: templates/layouts/default.jade [hunk 2]
```diff
@@ -108,21 +107,16 @@ html
...
 				span: a(href='/keystone/signin') Admin Login
 				br
 				p Powered by <a href="http://keystonejs.com" target="_blank">KeystoneJS</a>.
 			.col-md-4: .text-center
-				h3 Unofficial Tools
-				a(href='http://fafafaf.bitbucket.org/') Replay Parser
-			
+				h3 Connect
+				div.social
+				<a href="https://www.facebook.com/ForgedAllianceForever" title="Follow FAForever on Facebook" target="_blank"><i class="fa fa-facebook fa-3x"></i></a>
```
Comment (Blodir) [inline templates/layouts/default.jade]: add some padding to each icon

Comment (Raistlfiren) [inline templates/layouts/default.jade]: screen shot 2016-07-14 at 1 53 02 pm

Is that what you are looking for?

Comment (Blodir) [inline templates/layouts/default.jade]: Maybe a bit more padding-left/right, but it's looking quite okay :)

**Comment by Raistlfiren** [comment]
Okay - updated it now to reflect the changes you requested.

**Pull request merged**